server/src/db/migrations/20230809145512_createVotes.cjs

```diff
@@ -0,0 +1,24 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+    return knex.schema.createTable("votes", table => {
+        table.bigIncrements("id")
+        table.bigInteger("voterId").notNullable().unsigned().index().references("users.id")
+        table.bigInteger("reviewId").notNullable().unsigned().index().references("reviews.id")
+        table.integer("voteValue").notNullable()
```

Comment:
up to you, but I would just end up calling this column "value". 
fine to keep though if you want to save time